install_config/aggregate_logging.adoc

```diff
@@ -95,147 +77,274 @@ deployed. To control component placement, specify node selectors per component t
-this service account.
-====
+[cols="3,7",options="header"]
+|===
+|Parameter
+|Description
 
-. Enable the Elasticsearch service account to get cluster role bindings so that
-it can verify a user's roles and allow access to operations
-logs:
-+
-====
-----
-$ oadm policy add-cluster-role-to-user rolebinding-reader \
-     system:serviceaccount:logging:aggregated-logging-elasticsearch <1>
-----
-<1> Use the project you created earlier (for example, *logging*) when specifying
-this service account.
-====
+ifdef::openshift-origin[]
+|`*openshift_logging_image_prefix*`
+|The prefix for logging component images. For example, setting the prefix to
+*openshift/origin-* creates *openshift/origin-logging-deployer:v1.5*.
 
-[[aggregate-logging-specifying-deployer-parameters]]
-== Specifying Deployer Parameters
+|`*openshift_logging_image_version*`
+|The version for logging component images. For example, setting the version to
+*v1.2* creates *openshift/origin-logging-fluentd:v1.5*.
+endif::openshift-origin[]
+ifdef::openshift-enterprise[]
+|`*openshift_logging_image_prefix*`
+|The prefix for logging component images. For example, setting the prefix to
+*registry.access.redhat.com/openshift3/* creates *registry.access.redhat.com/openshift3/logging-fluentd:latest*.
 
-Parameters for the EFK deployment may be specified in the
-form of a xref:../dev_guide/configmaps.adoc#dev-guide-configmaps[ConfigMap],
-a xref:../dev_guide/secrets.adoc#dev-guide-secrets[secret],
-or template parameters (which are passed to the deployer in
-environment variables). The deployer looks for each value first in a
-*logging-deployer* ConfigMap, then a *logging-deployer* secret, then as
-an environment variable. Any or all may be omitted if not needed.
-If you are specifying values within a ConfigMap, the values will not be reflected
-in the output from `oc new-app`, but will still precede the corresponding
-template values within the deployer pod.
+|`*openshift_logging_image_version*`
+|The version for logging component images. For example, setting the version to
+*v3.5* creates *registry.access.redhat.com/openshift3/logging-fluentd:v3.5*.
+endif::openshift-enterprise[]
 
-The available parameters are outlined below. Typically, you should at
-least specify the host name at which Kibana should be exposed to client
-browsers, and also the master URL where client browsers will be directed to
-for authenticating to {product-title}.
+| `*openshift_logging_use_ops*`
+| If set to `true`, configures a second Elasticsearch cluster and Kibana for
+operations logs. Fluentd splits logs between the main cluster and a cluster reserved
+for operations logs (which consists of *_/var/log/messages_* on nodes and the logs from the
+projects *default*, *openshift*, and *openshift-infra*).  This means a second Elasticsearch
+and Kibana are deployed. The deployments are distinguishable by the *-ops* suffix included
+in their names and have parallel deployment options listed below.
 
```

Comment:
The role 'openshift_hosted_logging' will intentionally go away in 3.5 in favor of 'openshift_logging' per the direction from @detiber when we took upon the new work.